CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
-## UNRELEASED
+## 0.4.2 (November 22, 2021)
+FEATURES:
+* Support TLS and mutual TLS for the CTS API and CLI. [[GH-466](https://github.com/hashicorp/consul-terraform-sync/issues/466)]
+* **(Enterprise Only)** Add Terraform Cloud workspace tagging support to add, require, and restrict tags with new `driver.terraform-cloud.workspaces` options.
 
 IMPROVEMENTS:
 * **(Enterprise Only)** Add default address for the Terraform Cloud driver to https://app.terraform.io.
 
+BUG FIXES:
+* Fix Services Regex, Catalog Services, and Consul KV conditions to use Consul API blocking queries for more efficient monitoring of changes on Consul. [[GH-460](https://github.com/hashicorp/consul-terraform-sync/pull/460), [GH-467](https://github.com/hashicorp/consul-terraform-sync/pull/467)]
```

Comment:
Maybe this could be `Services` rather than `Services Regex` since that's how the condition is documented elsewhere?

```suggestion
* Fix Services, Catalog Services, and Consul KV conditions to use Consul API blocking queries for more efficient monitoring of changes on Consul. [[GH-460](https://github.com/hashicorp/consul-terraform-sync/pull/460), [GH-467](https://github.com/hashicorp/consul-terraform-sync/pull/467)]
```

Reply:
Yea, I get what you're saying - user-facing wise we don't use this language. I think the extra distinction here is that "Services condition" could refer to services condition configured via `task.services` and `condition "services"` and this fix I _believe_ is just for `condition "services"`.

What about:

> Fix conditions that are configured through Services, Catalog Services, and Consul KV condition blocks to use Consul API blocking queries for more efficient monitoring of changes on Consul. [[GH-460](https://github.com/hashicorp/consul-terraform-sync/pull/460), [GH-467](https://github.com/hashicorp/consul-terraform-sync/pull/467)]

Reply:
sounds good to me!

Reply:
The default Services condition was working as expected though. The bug and fix only affects the services condition when the regular expression is configured

Reply:
Gotcha, thanks for clarifying Kim. It sounds like having the "Regex" adds necessary details on the Services condition vs. naming a feature "Services Regex condition"

Reply:
Can we mention specifically that regex needs to be configured then to reproduce the issue?

How about: 

Fix for more efficient monitoring of changes on Consul when specifying conditions using Consul KV and Catalog Services condition blocks and when configuring regex using the Services condition block  [GH-460, GH-467]

Reply:
@wilkermichael and I just chatted about this and are now suggesting:

> Fix Services condition when configured with regex, Consul KV condition, and Catalog Services condition to use Consul API blocking queries for more efficient monitoring of changes on Consul. [GH-460, GH-467]

@findkim does this capture things ok to you?

Reply:
I'm liking these iterations! Yeah let's go with the latest one

Reply:
```suggestion
* Fix Services condition when configured with regex, Consul KV condition, and Catalog Services condition to use Consul API blocking queries for more efficient monitoring of changes on Consul. [[GH-460](https://github.com/hashicorp/consul-terraform-sync/pull/460), [GH-467](https://github.com/hashicorp/consul-terraform-sync/pull/467)]
```